playbooks/adhoc/s3_registry/s3_registry.yml

```diff
@@ -6,7 +6,7 @@
 # The AWS access/secret keys should be the keys of a separate user (not your main user), containing only the necessary S3 access role.
 # The 'clusterid' is the short name of your cluster.
 
-- hosts: tag_clusterid_{{ clusterid }}:&tag_host-type_openshift-master
+- hosts: tag_clusterid_{{ clusterid }}:&tag_host_type_openshift_master
```

Comment:
This one is a bit tricky, we have some users that have run this playbook that have the 'replace_dash_in_groups' variable in ec2.ini set to False, which means that this change would cause this script to no longer work for those users.

It might be better to add a variable for setting this instead.

That, or we would need to update the inventory/aws/hosts/ec2.ini file in this repo and all of the references with '-''s in them under playbooks/aws/